CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing to the Non-Fungible Token Standard
+
+The following is a set of guidelines for contributing to the Flow NFT standard. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```

Comment:
Should this be changed to say ramdom coin toss implementation instead of NFT standard?

Reply:
ha you're right! Good catch